### DISCUSSION TITLE
Can Explorer Patcher move the *Windows 11* taskbar to the top of the screen?

### DISCUSSION THREAD

**Discussion by CalculonPrime** [valinet/ExplorerPatcher#3385] [Q&A]
[I see a similar question was asked, answered, and closed, but that was when Windows 10 taskbar was still an option.  Looks like that's going to stop working.]

Was directed to Explorer Patcher as one possible way of moving the taskbar in Windows 11 to the top of the screen where it belongs.  However, I see some notes with the latest release that some future version of Windows 11 won't support the Windows 10 taskbar option in Explorer Patcher, and when I switch to the Windows 11 taskbar, I can't drag it to the top or see any config to change its position.

Is there any way to do it?  Are we all going to just give up, lean back, and enjoy the horror?

**Comment by pyrates999** [comment]
That is up to Microsoft to implement.  EP can only enable features for code that is there.

**Comment by CalculonPrime** [comment]
There was previously a registry setting to move the Windows 11 taskbar to the top (which no longer works), so I thought perhaps your patcher could flip some bit somewhere and trigger that, but I guess not.

**Comment by pyrates999** [comment]
If there was a way to do it, it would have been implemented.

**Comment by IgorTrifunovic** [comment]
I just pray to God people will be so pissed off by MS decisions they will start with massive contributions to open source OSes so we can all live free.

**Comment by Amrsatrio** [comment]
Not with EP. At some point Microsoft made the 11 taskbar disregard the registry value for the position because they knew that people have been doing it and decided to kill it off.

**Reply by snakeeyes021** [reply]
Is that what it does? Disregard the value? I tried that method (not knowing it had been patched), and it seemed like every time I would restart Explorer in order to implement the new registry value, the value would be reverted. I didn't look into it any further, but I assumed that's all it was doing, not ignoring the value, just changing it back.
